designer/server/src/main/scala/pl/touk/nussknacker/ui/db/timeseries/questdb/QuestDbFEStatisticsRepository.scala

```diff
@@ -82,6 +82,7 @@ private class QuestDbFEStatisticsRepository(private val engine: AtomicReference[
         name -> count
       }
       .toMap
+    logger.debug("Fetched FE statistics: {}", result)
```

Comment:
I'm not sure if we need it here. It seems that when we enable debug logging for a slick we will be able to see the same result. 

But if we see a need to have some better visibility of what happens in the statistics logic, I would consider adding some business-related debug logs at the `UsageStatisticsReportsSettingsService` level.

Reply:
I forgot it to remove